docs/github_pages/contributing/submitting_a_pr.md

```diff
@@ -86,7 +86,7 @@ cmake --build . -j <num jobs>   # invokes make (or ninja, etc)
 ctest
 ```
 
-See [CMake Options](./setup/cmake_options.md) for details on customizing the build. To
+See [CMake Options](../setup/cmake_options.md) for details on customizing the build. To
```

Comment:
This will still be a broken link on the new doc site: https://nvidia.github.io/thrust/contributing/submitting_a_pr.html

The link is generated to point to `.../cmake_options.md`, but the link needs to point at a `.../cmake_options.html`.

@brycelelbach Is there some doxybook magic we can use to convert the file extension in the link so that both the markdown and doxy versions will work?

Reply:
I'm not following. The current formulation of `.setup/cmake_options.md` ends up generating `https://nvidia.github.io/thrust/contributing/setup/cmake_options.md` because `submitting_a_pr.md` exists in `contributing/` so `./setup/cmake_options.md` points to `contributing/setup/cmake_options.md`. 

Wouldn't changing the relative path to `../setup` to reference the parent directory correct it?

Reply:
There is already something in Doxybook/Jekyll that should do this (rewrite `.md` to `.html`). It's one of the Jekyll modules, I think?

Reply:
@brycelelbach whatever is supposed to be rewriting `.md` to `.html` in URLs is broken, because it's not happening, and the link to CMake Options generates a 404 on thrust.github.io.